src/rp2_common/pico_lwip/tools/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Compile the http content into a source file "pico_fsdata.inc" in a format suitable for the lwip httpd server
+# Pass the target library name library type and the list of httpd content
+function(pico_lwip_httpd_content TARGET_LIB TARGET_TYPE)
```

Comment:
probably pico_add_ or pico_set_ depending on whether it can be called multiple times for the same target

Reply:
I think it can/should only be used once per target